spec/features/users_should_see_home_page_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe "users should see home page" do
+  context "after adding gif to db" do
+    it "shows gifs on page" do
+
+      Gif.create(image_path: 'https://media.giphy.com/media/1apNICVnC3AEo/giphy.gif')
+      Gif.create(image_path: 'https://media.giphy.com/media/yYSSBtDgbbRzq/giphy.gif')
+
+      visit root_path
+      expect(page).to have_content "All Gifs"
+      expect(page).to have_css("img[src*='https://media.giphy.com/media/1apNICVnC3AEo/giphy.gif']")
```

Comment:
Use a `within` block to assert that there is appropriate html/css on the page too